Specialization/Frost.lua

```diff
@@ -207,11 +194,11 @@ function DeathKnight:FrostGlowCooldowns()
 	local deathKnightRuneforgeRazorice = enchant[weaponRunes.Razorice];
 
 	local frostwyrmsFuryReady = DeathKnight.db.frostFrostwyrmsFuryAsCooldown and cooldown[FR.FrostwyrmsFury].ready;
-	local abominationLimbReady = DeathKnight.db.abominationLimbAsCooldown and covenantId == Necrolord and cooldown[FR.AbominationLimb].ready;
+	local abominationLimbReady = DeathKnight.db.abominationLimbAsCooldown and (covenantId == Necrolord or talents[COMMON.AbominationLimbTalent]) and cooldown[COMMON.AbominationLimb].ready;
 
 	if DeathKnight.db.alwaysGlowCooldowns then
 		MaxDps:GlowCooldown(FR.FrostwyrmsFury, frostwyrmsFuryReady);
-		MaxDps:GlowCooldown(FR.AbominationLimb, abominationLimbReady);
+		MaxDps:GlowCooldown(COMMON.AbominationLimb, abominationLimbReady);
```

Comment:
If you have the talent and are Necrolord, this is a problem right? Moreover, just remove the covenant stuff :P

Reply:
I'm pretty sure I tested this but will check again. I believe the talent has different ids but the spell doesn't